### ISSUE TITLE
Fix request: False positive in BufferFileStreaming

### ISSUE THREAD

**Issue by jborgers** [jborgers/PMD-jPinpoint-rules#272]
false positive:
```java
Properties props = new Properties();
try(InputStream fis = new FileInputStream(fileName);      // false violation on fis
	InputStream bis = new BufferedInputStream(fis)) {
	props.load(bis);
}
```
